tests/smoke.rs

```diff
@@ -337,3 +340,57 @@ fn test_inline(rafs_version: &str) {
     builder.build_inline_lower(rafs_version);
     builder.check_inline_layout();
 }
+
+#[test]
+fn integration_test_decompress() {
+    let mut prefix =
+        PathBuf::from(var("TEST_WORKDIR_PREFIX").expect("Please specify TEST_WORKDIR_PREFIX env"));
+    prefix.push("");
```

Comment:
Why `push("")` here?

Reply:
> Why `push("")` here?

A path with trailing slash is required by TempDir::new_with_prefix(). Prefix.push("") optionally appends a slash.

Reply:
> Why `push("")` here?

A comment is better.